source/includes/Patient/patients.md

```diff
@@ -136,86 +136,323 @@ DELETE /patient/:target_patient_uuid?purge=true
     **uuid** | `String` | uuid to delete
     *purge* | `Boolean` | The resource will be voided/retired unless purge = 'true'
 
+
 ## List patientIdentifier sub resources
 
-* ### List all patientIdentifier sub resources for a patient.
+> List patientIdentifier sub resources
 
-```console
+```shell
 GET /patient/:target_patient_uuid/identifier
 ```  
 
-    Retrieve all <b>identifier</b> sub resources of a <b>patient</b> resource by `target_patient_uuid`.Returns a `404 Not Found` status if patientIdentifier not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+```java
 
-* ### List patientIdentifier sub resource by it's UUID and parent patient UUID.
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+Request request = new Request.Builder()
+  .url("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/e739808f-f166-42ae-aaf3-8b3e8fa13fda/identifier")
+  .method("GET", null)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Cookie", "JSESSIONID=33B84B3BEA8E81E9D22D5A722815E010")
+  .build();
+Response response = client.newCall(request).execute();
 
-```console
+```
+
+```java
+
+var myHeaders = new Headers();
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Cookie", "JSESSIONID=33B84B3BEA8E81E9D22D5A722815E010");
+
+var requestOptions = {
+  method: 'GET',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+
+fetch("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/patient/e739808f-f166-42ae-aaf3-8b3e8fa13fda/identifier", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+  .catch(error => console.log('error', error));
+
+```
+
+> Success Response
+
+```response
+
+{
+    "results": [
+        {
```

Comment:
Feels like this one is not algines well correct me if am wrong

Reply:
ok @Akayeshmantha I will check once again for alingment